scripts/apidocgen/postprocess/main.go

```diff
@@ -0,0 +1,206 @@
+
+	err := os.RemoveAll(apiPath)
+	if err != nil {
+		return xerrors.Errorf(`os.RemoveAll failed for "%s": %w`, apiPath, err)
+	}
+
+	err = os.MkdirAll(apiPath, 0755)
+	if err != nil {
+		return xerrors.Errorf(`os.MkdirAll failed for "%s": %w`, apiPath, err)
+	}
+	return nil
+}
+
+func writeDocs(sections [][]byte) error {
+	log.Println("Write docs to destination")
+
+	apiDir := path.Join(docsDirectory, apiSubdir)
+	err := os.WriteFile(path.Join(apiDir, apiIndexFile), []byte(apiIndexContent), 0644) // #nosec
+	if err != nil {
+		return xerrors.Errorf(`can't write the index file: %w`, err)
+	}
+
+	type mdFile struct {
+		title string
+		path  string
+	}
+	var mdFiles []mdFile
+
+	// Write .md files for grouped API method (Templates, Workspaces, etc.)
+	for _, section := range sections {
+		sectionName, err := extractSectionName(section)
+		if err != nil {
+			return xerrors.Errorf("can't extract section name: %w", err)
+		}
+		log.Printf("Write section: %s", sectionName)
+
+		mdFilename := toMdFilename(sectionName)
+		docPath := path.Join(apiDir, mdFilename)
+		err = os.WriteFile(docPath, section, 0644) // #nosec
+		if err != nil {
+			return xerrors.Errorf(`can't write doc file "%s": %w`, docPath, err)
+		}
+		mdFiles = append(mdFiles, mdFile{
+			title: sectionName,
+			path:  "./docs/" + mdFilename,
+		})
+	}
+
+	// Update manifest.json
+	type route struct {
+		Title       string  `json:"title,omitempty"`
+		Description string  `json:"description,omitempty"`
+		Path        string  `json:"path,omitempty"`
+		IconPath    string  `json:"icon_path,omitempty"`
+		State       string  `json:"state,omitempty"`
+		Children    []route `json:"children,omitempty"`
+	}
+
+	type manifest struct {
+		Versions []string `json:"versions,omitempty"`
+		Routes   []route  `json:"routes,omitempty"`
+	}
+
+	manifestPath := path.Join(docsDirectory, "manifest.json")
+	manifestFile, err := os.ReadFile(manifestPath)
+	if err != nil {
+		return xerrors.Errorf("can't read manifest file: %w", err)
+	}
+	log.Printf("Read manifest file: %dB", len(manifestFile))
+
+	var m manifest
+	err = json.Unmarshal(manifestFile, &m)
+	if err != nil {
+		return xerrors.Errorf("json.Unmarshal failed: %w", err)
+	}
+
+	for i, r := range m.Routes {
+		if r.Title != "API" {
+			continue
+		}
+
+		var children []route
+		for _, mdf := range mdFiles {
+			docRoute := route{
+				Title: mdf.title,
+				Path:  mdf.path,
+			}
+			children = append(children, docRoute)
+		}
+
+		m.Routes[i].Children = children
+		break
+	}
+
+	manifestFile, err = json.MarshalIndent(m, "", "  ")
+	if err != nil {
+		return xerrors.Errorf("json.Marshal failed: %w", err)
+	}
+
+	err = os.WriteFile(manifestPath, manifestFile, 0644) // #nosec
+	if err != nil {
+		return xerrors.Errorf("can't write manifest file: %w", err)
+	}
+	log.Printf("Write manifest file: %dB", len(manifestFile))
+	return nil
+}
+
+func extractSectionName(section []byte) (string, error) {
+	scanner := bufio.NewScanner(bytes.NewBuffer(section))
```

Comment:
Minor thing, but a better fit 😄.

```suggestion
	scanner := bufio.NewScanner(bytes.NewReader(section))
```